README.md

```diff
@@ -16,7 +16,7 @@ Project Status:
 | [Evolution](https://greenelab.github.io/covid19-review/#evolutionary-and-genomic-analysis-of-sars-cov-2) | Evolutionary Perspectives on SARS-CoV-2 | #867 | In prep | In prep for mSystems
 | [Diagnostics](https://greenelab.github.io/covid19-review/#diagnostics) | Molecular and Serologic Diagnostic Technologies for SARS-CoV-2| ~#818~ #1094 | Revision in progress | _mSystems_ | [Release](https://github.com/greenelab/covid19-review/releases/tag/diagnostics-v1); [Preprint](https://arxiv.org/abs/2204.12598) |
 | [Pharmaceuticals](https://greenelab.github.io/covid19-review/#identification-and-development-of-therapeutics-for-covid-19) | Identification and Development of Therapeutics for COVID-19 | ✔️ | [Published](https://doi.org/10.1128/mSystems.00233-21) 🎉 | _mSystems_ | [Release](https://github.com/greenelab/covid19-review/releases/tag/pharmaceuticals-v3); [Preprint](http://arxiv.org/abs/2103.02723) |
-| [Vaccines- Established Platforms](https://greenelab.github.io/covid19-review/#application-of-traditional-vaccine-development-strategies-to-sars-cov-2) | Application of Traditional Vaccine Development Strategies to SARS-CoV-2 | #830 | Submitted | Submitted to _mSystems_ | [Release](https://github.com/greenelab/covid19-review/releases/tag/vaccines-trad-v1); [Preprint](https://arxiv.org/abs/2208.08907) |
+| [Vaccines- Established Platforms](https://greenelab.github.io/covid19-review/#application-of-traditional-vaccine-development-strategies-to-sars-cov-2) | Application of Traditional Vaccine Development Strategies to SARS-CoV-2 | #830 | Accepted 🎉  | _mSystems_ | [Release](https://github.com/greenelab/covid19-review/releases/tag/vaccines-trad-v1); [Preprint](https://arxiv.org/abs/2208.08907) |
```

Comment:
```suggestion
| [Vaccines- Established Platforms](https://greenelab.github.io/covid19-review/#application-of-traditional-vaccine-development-strategies-to-sars-cov-2) | Application of Traditional Vaccine Development Strategies to SARS-CoV-2 | #830 | Accepted 🎉  | _mSystems_ | [Release](https://github.com/greenelab/covid19-review/releases/tag/vaccines-trad-v2.1); [Preprint](https://arxiv.org/abs/2208.08907) |
```
I'll call the corrected v2 release v2.1 so I don't have to delete my early v2. v2.1 will be the commit made after we merge this pull request.